commands/redis/redis-flush

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#ddev-generated
+## Description: Run redis-cli inside the redis container
+## Usage: redis-flush
+## Example: "redis-flush"
+
+redis-cli -a redis --no-auth-warning FLUSHALL ASYNC
```

Comment:
```suggestion
## Description: Clear all Redis databases
## Usage: redis-flush
## Example: "ddev redis-flush"

set -x
redis-cli -a redis --no-auth-warning FLUSHALL ASYNC
```

Fixed the command description and example.
Redis commands are so complicated, let's add `set -x` to see what is actually executing.